exercises/simple-linked-list/src/example.c

```diff
@@ -0,0 +1,127 @@
+#include "simple_linked_list.h"
+
+/*
+Initialize new node
+*/ 
+    node * new_node(int data) 
+{
+   
node * startp = (node *) malloc(sizeof(node));
+   
if (startp == NULL) {
+      
printf("error in making new node");
+      
return NULL;
+   
}
+   
startp->data = data;
+   
startp->next = NULL;
+   
return startp;
+
}
+
+
 
+/*
+Find node by key (-1 otherwise)
+*/ 
+int find_node(node * startp, int key) 
+{
+   
int result = -1;
+   
while (startp != NULL) {
+      
if (startp->data == key)
+         
return startp->data;
+      
startp = startp->next;
+   
}
+   
return result;
+
}
+
+
 
+/*
+Insert data at the front of the list
+*/ 
+void insert_beg(node ** startp, int data_new) 

```

Comment:
This function name isn't great, specifically the `beg` part. Can you use a different one? Maybe `head_insert()` or something similar